bool toggle
---
bool april_detected